classes/plugins/PubObjectsExportPlugin.php

```diff
@@ -397,33 +411,19 @@ public function getRepresentationFilter()
     }
 
     /**
-     * Get status names for the filter search option.
+     * Get action names.
      *
-     * @return array (string status => string text)
+     * @return array (string action => string text)
      */
-    public function getStatusNames()
+    public function getExportActionNames()
```

Comment:
Add return type? (And in DOAJExportPlugin)

Reply:
This is actually the old function, I just resorted the functions, so I have not change anything, but I can add the return type -- it seems no other plugin uses it currently...